packages/office-addin-debugging/src/usagedata-helper.ts

```diff
@@ -0,0 +1,21 @@
+import * as defaults from './defaults';
+import * as usageData from "office-addin-usage-data";
+const usageDataObject: usageData.OfficeAddinUsageData = new usageData.OfficeAddinUsageData(defaults.usageDataOptions);
+
+export function sendUsageDataSuccessEvent(method: string) {
+    const usageDataInfo = {
+        Method: [method],
+        Platform: [process.platform],
+        NodeVersion: [process.version],
+        Succeeded: [true]
+    }
+    usageDataObject.reportEvent(defaults.usageDataProjectName, usageDataInfo);
+}
+
+export function sendUsageDataCustomEvent(usageDataInfo: Object) {
```

Comment:
This should change to what the helper method is doing. Take some parameters along with an object that can supply the additional parameters and then have the implementation build up the object needed to pass to appinsights.